code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -27,13 +34,17 @@
 	mine_range = 5
 
 /obj/item/projectile/plasma/adv/mech
-	damage = 10
+	damage = 40
 	range = 9
 	mine_range = 3
-
+	dismemberment = 20
+	pressure_check_scaling = TRUE

```

Comment:
also, cobby  set this to false in his recommended code, but I set it to true. I believe it was a typo.